Test/com/nicoles_change_maker/test/CoinMapTest.java

```diff
@@ -0,0 +1,39 @@
+package com.nicoles_change_maker.test;
+import com.nicoles_change_maker.CoinMap;
+import org.junit.*;
+import java.util.HashMap;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by bears8yourface on 1/5/16.
+ */
+public class CoinMapTest {
+    CoinMap coinMap = new CoinMap();
+    HashMap dictionary = coinMap.dictionary;
+    private static final double DELTA = .001;
+
+    @Test
+    public void testPennyValue() {
+        assertEquals(.01, dictionary.get("P"));
```

Comment:
Here's an example of an object violating encapsulation and exposing its internals. Instead of calling a method on `CoinMap`, e.g.: 

```java
assertEquals(.01, CoinMap.valueForCoin("P"));
```

I have to reach inside the object and pull out `dictionary`, a data structure. This class would be better encapsulated if it hid the `HashMap` internally, exposing the behavior I want ("tell me what the value of this coin is") without exposing the implementation ("look up the value of this coin in a `HashMap`)".